js/index.js

```diff
@@ -1,42 +1,107 @@
+        "nav-item-5": "About",
+        "nav-item-6": "Contact",
+        "img-src": "img/logo.png"
+    },
+    "cta": {
+        "h1": "DOM Is Awesome",
+        "button": "Get Started",
+        "img-src": "img/header-img.png"
+    },
+    "main-content": {
+        "features-h4": "Features",
+        "features-content": "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "about-h4": "About",
+        "about-content": "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "middle-img-src": "img/mid-page-accent.jpg",
+        "services-h4": "Services",
+        "services-content": "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "product-h4": "Product",
+        "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "vision-h4": "Vision",
+        "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    },
+    "contact": {
+        "contact-h4": "Contact",
+        "address": "123 Way 456 Street Somewhere, USA",
+        "phone": "1 (888) 888-8888",
+        "email": "sales@greatidea.io",
+    },
+    "footer": {
+        "copyright": "Copyright Great Idea! 2018"
+    },
+    "stretch": {
+        "img-lambda": "img/lambda.png"
+    }
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// Updating the nav menu
+var add = document.createElement('a')
+document.querySelector("nav").appendChild(add);
+let nav_items = document.querySelectorAll('a');
+for (let i = 0; i < nav_items.length; i++) {
+    nav_items[i].textContent = siteContent["nav"]["nav-item-" + i];
+    nav_items[i].style.color = 'green'
+}
+// Updating Class "cta section"
+let heading = document.querySelector('.cta-text h1');
+let heas = siteContent["cta"]["h1"];
+var newArray = heas.split(" ")
+for (let i = newArray.length - 1; i > -1; i--) {
+    var node = document.createElement("h1"); // Create a <li> node
+    var textnode = document.createTextNode(newArray[i]); // Create a text node
+    node.appendChild(textnode); // Append the text to <li>
+    document.querySelector(".cta-text").prepend(node);
```

Comment:
Minor tweak - I would create a variable outside the for loop for this querySelector.

Right now, the code is running `document.querySelector(".cta-text")` each time it loops through this block. If we pull the querySelector outside the for loop, the code will only need to check the document once, an then use the variable inside the block here.